packages/actionmenu/src/react/ellipsis.tsx

```diff
@@ -1,11 +1,11 @@
 import { css } from 'glamor'
-import React, { forwardRef } from 'react'
+import React, { HTMLAttributes, forwardRef } from 'react'
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
```

Comment:
i think we can get rid of filterReactProp in most places because the typings should tell us when unsupported props are passed. there might be a couple places that's not the case...but snapshots should alert us and we can remove the props without the filterReactProps helper